src/greeter/GreeterApp.cpp

```diff
@@ -134,6 +155,39 @@ namespace SDDM {
         connect(this, &GreeterApp::primaryScreenChanged, this, [this](QScreen *) {
             activatePrimary();
         });
+
+        // create filewatcher for adminInfo-file
+        QString adminInfoPath = SDDM::mainConfig.AdminInfoPath.get();
+
+        // check if config has been set
+        if (!adminInfoPath.isEmpty()) {
+            // check for theme-compatibility
+            if (m_metadata->adminInfoSupport()) {
+                m_fileAgent = new QFileSystemWatcher(this);
+
+                // connect signals
+                connect(m_fileAgent, &QFileSystemWatcher::fileChanged, this, &GreeterApp::fileChanged);
```

Comment:
Also: yes, we have a theme updated with support for this feature. But keep in mind that we are not artistic when it comes to design. (It was only for testing purposes after all)

https://github.com/preisi/cip-theme/tree/adminInfo

Furthermore, the HTML is just a feature provided by the "Text QML Type". So no need to use it, normal text works, too.